deletefb/deletefb.py

```diff
@@ -30,7 +41,7 @@ def run_delete():
     parser.add_argument(
         "-E",
         "--email",
-        required=True,
+        required='--help-env' not in sys.argv,
```

Comment:
I think there might be a better way to make it so that `--help-env` doesn't require these. I'll have to look at the argparse docs again though.